source/playbooks/CIS/lambda/cis116.py

```diff
@@ -190,19 +193,46 @@ def failed():
                         PolicyArn=policy_arn
                     )
 
-        response = iam.add_user_to_group(
+        iam.add_user_to_group(
             GroupName=group_name,
             UserName=username
         )
 
-        LOGGER.debug(response)
-
         message['State'] = 'RESOLVED'
-        message['Note'] = '' # use default
+        message['Note'] = ''  # use default
         notify(finding, message, LOGGER, cwlogs=APPLOGGER, sns=AWS)
         return
 
     except Exception as e:
         LOGGER.error(e)
         failed()
         return
+
+
+def does_group_exist(iam, group_name):
```

Comment:
This function should go above where it is called from or you will get a `NameError` not found when you try to call it on line 140.

Reply:
I doubt if this is the case, as in Python you can define in any order. did you confuse with JS? I tested both locally and playbook actions.